examples/v1alpha1/cron/triggerbinding.yaml

```diff
@@ -5,6 +5,6 @@ metadata:
 spec:
   params:
   - name: gitrevision
-    value: master
+    value: main
```

Comment:
In this demo, cloning the `master` branch will fail because it has been renamed to `main`.